src/Frontend/src/stores/ThroughputStore.ts

```diff
@@ -24,14 +24,14 @@ export const useThroughputStore = defineStore("ThroughputStore", () => {
     }
 
     if (!testResults.value?.audit_connection_result.connection_successful) {
```

Comment:
If we have both audit and monitoring enabled, and one of them has a connection error, then I think we should display the error icon.
So this would check for both:

```suggestion
    if (!testResults.value?.audit_connection_result.connection_successful || (useIsMonitoringEnabled() && !testResults.value?.monitoring_connection_result.connection_successful)) {
```

Then you can delete the last check from line 30 I think?

Reply:
You can ignore - it's just a refactor of what you had.